app/src/main/java/com/example/intent/MainActivity.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.intent
+
+import android.content.Intent
+import android.net.Uri
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import androidx.core.app.ShareCompat
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var edWebsite: EditText
```

Comment:
better use nullable properties without lateinit keyword: `private var edWebsite: EditText? = null`
Or best way - use viewbinding instead of findViewById